src/app/modules/workspace/workspace.service.ts

```diff
@@ -83,14 +84,65 @@ export const updateWorkspaceTitle = ({
   )
 }
 
-export const deleteWorkspace = (
-  workspaceId: string,
-  shouldDeleteForms: boolean,
-): ResultAsync<any, DatabaseError> => {
-  return okAsync({
-    workspaceId: workspaceId,
-    shouldDeleteForms: shouldDeleteForms,
+export const deleteWorkspace = ({
+  workspaceId,
+  shouldDeleteForms,
+}: {
+  workspaceId: string
+  shouldDeleteForms: boolean
+}): ResultAsync<void, DatabaseError> => {
```

Comment:
this should be `PossibleDatabaseError` as per the return type of `transformMongoError` right?

Reply:
Yes, updated the type to `PossibleDatabaseError`